src/Example.bas

```diff
@@ -2,6 +2,15 @@ Attribute VB_Name = "Example"
 '@Folder MVVM.Example
 Option Explicit
 
+'VF: Windows 10 is having a hard time handling multiple monitors, especially if different resolutions and more so if legacy applications like the VBE
+    'keeps shrinking the userform in the VBE and thus showing the shrunk form <- must counteract this ugly Windows bug by specifying Height and Width of IView
+    'rendering engine was changed from 2010 to 2013
+    'should go into IView, shouldn't it?

```

Comment:
Yeah `IView` makes sense for that, although the intent is to eventually get layout/container panels to actually do the work, and they'd probably be fine working with the `UserForm` type directly, to handle resizing events.